src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -395,4 +395,24 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
         this.hasInvalidBatchFields = false;
         this.missingBatchHeaderFieldLabels = [];
     }
+
```

Comment:
This comment is more of an open question/conversation. What're your thoughts around adding jsdocs when these qa locator getters start and end? I personally like them if only to immediately call out the group when we're strolling through these components in the future.